src/containers/Board/board.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { makeStyles } from '@mui/styles';
+
+const useBoardStyles = makeStyles(() => ({
+  board: {
+    padding: '1rem',
+  },
+  board_header: {
+    display: 'flex',
+    backgroundColor: '#fff',
```

Comment:
a lot of variables are duplicated. you can either use a theme or move it to theme/colors files for example.